e2e/suites/actions/create/create-file-from-template.test.ts

```diff
@@ -111,7 +111,7 @@ describe('Create file from template', () => {
   };
   let link: string;
 
-  beforeAll(async () => {
+  beforeAll(async (done) => {
```

Comment:
why is this "done" needed?
I see some inconsistencies, in some places you are removing the "done", in other places you're adding it. Why?

Reply:
I'm adding `done` here because we're using both: API calls and UI actions (`loginPage.loginWith(username)`) in the same `beforeAll()`.